src/EmbeddedManifold.jl

```diff
@@ -97,7 +97,7 @@ end
 
 function allocate_result(M::AbstractEmbeddedManifold, f::typeof(embed), x...)
     T = allocate_result_type(M, f, x)
-    return allocate(x[end], T, representation_size(decorated_manifold(M)))
```

Comment:
This is for sure the syntactically correct way (though they might do the same as it is currently implemented

Reply:
Actually after that commit they don't do the same thing ;).